library/src/main/java/com/readystatesoftware/chuck/api/ChuckInterceptor.java

```diff
@@ -73,6 +80,14 @@ public ChuckInterceptor maxContentLength(long max) {
         return this;
     }
 
+    public ChuckInterceptor redactHeader(String name) {
```

Comment:
Some developers may need to redact multiple headers, we should have this method instead:
`public ChuckInterceptor redactHeaders(String names...) {`

Reply:
Done